src/Cake.Core/Tooling/ToolSettings.cs

```diff
@@ -12,5 +13,11 @@ public class ToolSettings
         /// </summary>
         /// <value>The tool path.</value>
         public FilePath ToolPath { get; set; }
+
+        /// <summary>
+        /// Gets or sets the arguments.
+        /// </summary>
+        /// <value>The arguments.</value>
+        public Action<ProcessArgumentBuilder> ArgumentCustomization { get; set; }
```

Comment:
We could add an extension method for this, but just providing an `Action` is probably simpler and fits the most common use case where you want to add an argument that is missing.

If you completely want to rewrite the arguments, you could simply use the `builder.Clear()` method before adding your arguments.